internal/pkg/scaffold/go_mod_test.go

```diff
@@ -89,4 +89,6 @@ replace (
 	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
 	k8s.io/kube-state-metrics => k8s.io/kube-state-metrics v1.6.0
 )
+
+replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
```

Comment:
Why is this now needed but not before?

Reply:
We [pin](https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md#1-create-a-pr-for-release-version-and-changelogmd-updates) the exact SDK version for releases.